improver_tests/wind_calculations/wind_downscaling/test_RoughnessCorrection.py

```diff
@@ -194,45 +143,30 @@ def __init__(
                 Model orography field on post-processing grid
 
         """
-        if isinstance(nx_ny, int):
-            n_x = n_y = nx_ny
-        else:
-            n_x = nx_ny[0]
-            n_y = nx_ny[1]
-        self.n_x = n_x
-        self.n_y = n_y
+        self.shape = shape
         if AoS is None:
-            AoS = np.ones([n_x, n_y]) * 0.2
+            AoS = np.full(shape, 0.2, dtype=np.float32)
         if Sigma is None:
-            Sigma = np.ones([n_x, n_y]) * 20.0
+            Sigma = np.full(shape, 20.0, dtype=np.float32)
         if pporog is None:
-            pporog = np.ones([n_x, n_y]) * 250.0
+            pporog = np.full(shape, 250.0, dtype=np.float32)
         if modelorog is None:
-            modelorog = np.ones([n_x, n_y]) * 230.0
+            modelorog = np.full(shape, 230.0, dtype=np.float32)
         self.w_cube = None
-        self.aos_cube = set_up_cube(
-            1, [n_x, n_y], 1, data=AoS, height=0, name=None, unit=1
-        )
-        self.s_cube = set_up_cube(
-            1, [n_x, n_y], 1, data=Sigma, height=0, name=None, unit="m"
+        self.aos_cube = make_ancil_cube(AoS, None, 1, shape=shape)
+        self.s_cube = make_ancil_cube(
+            Sigma, "standard_deviation_of_orography_height", "m", shape=shape
         )
         if z_0 is None:
             self.z0_cube = None
         elif isinstance(z_0, float):
-            z_0 = np.ones([n_x, n_y]) * z_0
-            self.z0_cube = set_up_cube(
-                1, [n_x, n_y], 1, data=z_0, height=0, name=None, unit="m"
-            )
+            z_0 = np.full(shape, z_0, dtype=np.float32)
+            self.z0_cube = make_ancil_cube(z_0, None, "m")
         elif isinstance(z_0, list):
-            z_0 = np.array(z_0)
-            self.z0_cube = set_up_cube(
-                1, [n_x, n_y], 1, data=z_0, height=0, name=None, unit="m"
-            )
-        self.poro_cube = set_up_cube(
-            1, [n_x, n_y], 1, data=pporog, height=0, name=None, unit="m"
-        )
-        self.moro_cube = set_up_cube(
-            1, [n_x, n_y], 1, data=modelorog, height=0, name=None, unit="m"
+            self.z0_cube = make_ancil_cube(np.array(z_0), None, "m", shape=shape)
+        self.poro_cube = make_ancil_cube(pporog, "orography_height", "m", shape=shape)
+        self.moro_cube = make_ancil_cube(
+            modelorog, "orography_height", "m", shape=shape
```

Comment:
Our ancillaries call these cubes `"surface_altitude"`. Can we change these to be consistent?